atomicapp/nulecule/main.py

```diff
@@ -118,7 +117,7 @@ def __init__(self, app_spec, destination=None,
 
         # Process answers.
         self.answers_file = answers_file
-        self._process_answers()
+        self.config = self._process_answers(cli_answers)
```

Comment:
I think the right approach is to not call process_answers here but only in run/stop.
Here we would call `self.config=Config(cli=cli_answers)` and then later in `_process_answers()` we would call a new function from the config class to update answers like: `self.config.update_source(source='answers', data=answers)` or something along those lines.